tls/s2n_server_cert_request.c

```diff
@@ -17,6 +17,7 @@
 
```

Comment:
by convention of other files, s2n_server_* usually means extension sent by the server. i guess it is confusing here because the server is sending a client cert request which would otherwise be called `s2n_server_client_cert_request`

Reply:
correct. and the message is actually referred to just as "certificate request" so it didn't make sense with our naming conventions